ga4gh/cli.py

```diff
@@ -1123,12 +1123,32 @@ def configtest_main(parser=None):
 ##############################################################################
 
 
+def getRawInput(displayString):
+    userResponse = raw_input(displayString)
+    return userResponse
+
+
 class AbstractRepoCommandRunner(object):
 
     def __init__(self, args):
         self.args = args
         self.repoPath = args.repoPath
         self.repoManager = repo_manager.RepoManager(self.repoPath)
+        if 'force' in args:
```

Comment:
That would error for all the commands that did not have a `force` argument, which is most of them ... the error is "no force found in args" or something